proto/skaffold.proto

```diff
@@ -310,7 +358,7 @@ enum ErrorCode {
     // No Error codes
```

Comment:
I hit _Submit_ by accident.  One half-baked thought — which I think you're already considering — is that some of these ErrorCodes really identify the point of impact or result (e.g., the file-sync init error, or the dev-loop registeration errors), but sometimes not the actual error.

Reply:
I am going to add the `EndEvent` in another PR. Turns out this approach does not work with IDEs